tendrl/node_agent/manager/manager.py

```diff
@@ -239,6 +246,31 @@ def on_pull(self, raw_data):
                     self.etcd_client.write(("nodes/%s/Disks/free/%s") % (
                         raw_data["node_id"], disk), "")
 
+    def load_and_execute_discovery_plugins(self, node_id):
```

Comment:
Check https://github.com/Tendrl/node-agent/pull/134. I have done the suggested changes here.